internal/captain/values.go

```diff
@@ -232,6 +231,7 @@ func (p *PackagesValue) Type() string {
 type TimeValue struct {
 	raw  string
 	Time *time.Time
+	Now  bool
```

Comment:
Nit: Should make this a method rather than a public property, so it's compatible with interfaces.